lib/text/ttml_text_parser.js

```diff
@@ -1243,12 +1250,14 @@ shaka.text.TtmlTextParser.styleNs_ = 'http://www.w3.org/ns/ttml#styling';
 shaka.text.TtmlTextParser.styleEbuttsNs_ = 'urn:ebu:tt:style';
 
 /**
- * The namespace URL for SMPTE fields.
- * @const {string}
+ * The supported namespace URLs for SMPTE fields.
+ * @const !{Array.<{string}>}
```

Comment:
Style nit: {!Array.<string>}

Looks good otherwise. Thank you!

Reply:
doh!